app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -60,6 +60,13 @@ def promote
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_view_versions/:id/republish_repositories", N_("Forces a republish of the version's repositories' metadata.")
+    param :id, :identifier, :desc => N_("Content view version identifier"), :required => true
+    def republish_repositories
+      task = async_task(::Actions::BulkAction, ::Actions::Katello::Repository::MetadataGenerate, @version.repositories, :force => true)
```

Comment:
This can be handled separately from the PR; however, would it be useful to update the MetadataGenerate action to include more information in the task summary (e.g. include repo name, label or id)?  With the new action, the user may see a lot of actions generated, but looking at the task list from Monitor -> Tasks, they need to click in to each one and then to they dynflow console to see the repo that was affected.

E.g. in my case, there were 4 tasks, with the following 'Action'' column on Monitor -> Tasks:
- Metadata generate metadata generate; {"services_checked"=>["pulp", "pulp_auth...
- Metadata generate {"services_checked"=>["pulp", "pulp_auth"], "locale"=>"en"}
- Metadata generate {"services_checked"=>["pulp", "pulp_auth"], "locale"=>"en"}
- Metadata generate {"services_checked"=>["pulp", "pulp_auth"], "locale"=>"en"}